internal/core/services/scheduler/manager/scheduler_manager.go

```diff
@@ -20,25 +20,25 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
-package scheduler_manager
+package manager
 
 import (
 	"context"
 	"errors"
 	"fmt"
 
-	"github.com/topfreegames/maestro/internal/core/operations/deletescheduler"
+	"github.com/topfreegames/maestro/internal/core/operations/scheduler/create"
+	"github.com/topfreegames/maestro/internal/core/operations/scheduler/delete"
+	"github.com/topfreegames/maestro/internal/core/operations/scheduler/version/new"
+	_switch "github.com/topfreegames/maestro/internal/core/operations/scheduler/version/switch"
 
-	"github.com/topfreegames/maestro/internal/core/entities/game_room"
-	"github.com/topfreegames/maestro/internal/core/logs"
-	"github.com/topfreegames/maestro/internal/core/operations/newschedulerversion"
-	"github.com/topfreegames/maestro/internal/core/operations/switch_active_version"
-	"github.com/topfreegames/maestro/internal/core/services/scheduler_manager/patch_scheduler"
+	"github.com/topfreegames/maestro/internal/core/services/scheduler/manager/patch"
 
 	"github.com/topfreegames/maestro/internal/core/entities"
+	"github.com/topfreegames/maestro/internal/core/entities/game_room"
```

Comment:
`game_room` keeps the underline inside the package name, what do you think to rename this package as well?